doc/html/topics_id.html

```diff
@@ -0,0 +1,50 @@
+<html>
+<head>
+	<title>Konten Bantuan</title>
+</head>
+<body>
+
+	<!-- navigation header start -->
+	<table width="100%">
+	   <tr>
+	      <!-- back --><td width="50px">&nbsp;</td>
+	      <!-- home --><td style="text-align:center;"><b>Liferea Help</b></td>
+	      <!-- next --><td width="50px;" style="text-align:right;">&nbsp;</td>
+	   </tr>
+	</table>
+	<hr />
+	<!-- navigation header end -->
+
+<p>
+Dokumentasi pengguna ini berlaku untuk Liferea 1.12.
+</p>
+
+<h3>Topik Bantuan</h3>
+
+<ul>
+<li><a href="about_id.html">Perkenalan</a></li>
+<li><a href="concepts_id.html">Konsep Dasar</a></li>
+<li><a href="subscriptions_id.html">Mengelola Langganan</a></li>
+<li><a href="headlines_id.html">Mengelola Berita Utama</a></li>
+<li><a href="folders_id.html">Mengelola Folder</a></li>
+<li><a href="updating_id.html">Memperbarui Langganan</a></li>
+<li><a href="preferences_id.html">Pengaturan Preferensi</a></li>
+<li><a href="searching_id.html">Pencarian Berita Utama</a></li>
+<li><a href="enclosures_id.html">Lampiran/Podcasting</a></li>
+<li><a href="newsbin_id.html">Mengoleksi Item dalam Kotak Sampah Berita</a></li>
+<li><a href="onlineservices_id.html">Berlangganan ke Layanan Daring</a></li>
```

Comment:
The <a> tags above do reference files not included in this pull request.

It is ok not to translate them, but in this case we need to reference the English ones...